code/modules/cargo/exports/large_objects.dm

```diff
@@ -139,5 +139,5 @@
 	worth += gases[/datum/gas/tritium][MOLES]*5
 	worth += gases[/datum/gas/pluoxium][MOLES]*5
 	worth += gases[/datum/gas/freon][MOLES]*15
-	worth += gases[/datum/gas/hydrogen][MOLES]*25
+	worth += gases[/datum/gas/hydrogen][MOLES]*10
```

Comment:
Silent hydrogen debuf?

Reply:
uh no? i wrote that in the changelog too

Reply:
Ups, i lost it.